src/js/components/Select/index.d.ts

```diff
@@ -53,6 +60,7 @@ export interface SelectProps {
     | { key: string; reduce?: boolean }
     | ((...args: any[]) => any);
   emptySearchMessage?: string;
+  searching?: boolean;
```

Comment:
I  do not see a prop called `searching` on Select.

Reply:
it's set as boolean in the [CustomSearch](https://github.com/grommet/grommet/blob/master/src/js/components/Select/stories/CustomSearch.js) story